Mage/src/main/java/mage/abilities/dynamicvalue/common/CreaturesDiedThisTurnCount.java

```diff
@@ -13,6 +15,12 @@
 public enum CreaturesDiedThisTurnCount implements DynamicValue {
```

Comment:
please add a warning in the comments above that `CreaturesDiedWatcher` should be added as watcher for the `DynamicValue` to work.